src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/Models/Discount.cs

```diff
@@ -15,9 +16,11 @@ public class Discount : UpsertDiscountRequest
 
         public int? SubscriptionsTotal { get; set; }
 
+        public int? UsersSubscriptionTotal { get; set; }
+
         public List<string> FavoritesUsersId { get; set; }
 
-        public List<string> SubscriptionsUsersId { get; set; }
```

Comment:
Why you are delete SubscriptionUserId? It still can be used for other kind of discount